src/components/data_search/DatasetFilterConstants.tsx

```diff
@@ -24,5 +24,5 @@ export const defaultFilters = (datasets: DatasetTerm[]): FiltersTypes => {
 export const generateDefaultParticipantCountValues = (datasets: DatasetTerm[]) => datasets.reduce((acc, dataset) => {
   return {
     max: Math.max(acc.max, dataset.participantCount ? dataset.participantCount : 0),
-    min: Math.min(acc.min, dataset.participantCount ? dataset.participantCount : Infinity) };
+    min: 0 };
```

Comment:
Is this because some counts are undefined? Otherwise why wouldn't you want to use the lowest possible value?

If you only reduce over `max`, you should change the code to be explicit about that
```
    return { max: (datasets: DatasetTerm[]) => datasets.reduce((max, dataset) => Math.max(max, dataset.participantCount ? dataset.participantCount : 0), 0), min: 0 };
```

Reply:
> Is this because some counts are undefined? Otherwise why wouldn't you want to use the lowest possible value?

I'm not sure why it's failing to reduce down to 0, there are datasets with a populated value of 0, but it's only reducing down to 1.

I like the idea of simplifying the return.

Reply:
> Is this because some counts are undefined? Otherwise why wouldn't you want to use the lowest possible value?

I figured this out. When the count is `0`, the inner ternary is falsy, so it always returns `Infinity` instead of `0`

```
console.log(Math.min(Infinity, 0 ? 0 : Infinity));
```
yields `Infinity`

Reply:
OK, so if you wanted to preserve the current behavior, using `??` instead of ?:` would be a fix
```
export const generateDefaultParticipantCountValues = (datasets: DatasetTerm[]) => datasets.reduce((acc, dataset) => {
  return {
    max: Math.max(acc.max, dataset.participantCount ?? 0),
    min: Math.min(acc.min, dataset.participantCount ?? Infinity) };
}, {max: 0, min: Infinity});
```

Although typescript doesn't think `participantCount` can be `undefined`, so maybe this logic isn't necessary?